core/src/interface.d.ts

```diff
@@ -35,8 +35,7 @@ export * from './utils/overlays-interface';
 export * from './global/config';
 export { Gesture, GestureDetail } from './utils/gesture';
 
-// Global aux types
-export type TextFieldTypes = 'date' | 'email' | 'number' | 'password' | 'search' | 'tel' | 'text' | 'url' | 'time';
+export type TextFieldTypes = 'date' | 'email' | 'number' | 'password' | 'search' | 'tel' | 'text' | 'url' | 'time' | 'textarea';
```

Comment:
I would say remove `textarea` from here and add it as a separate option for `AlertInput`:

```
type?: TextFieldTypes | 'checkbox' | 'radio' | 'textarea';
```